packages/cli/src/ActiveExecutions.ts

```diff
@@ -179,7 +181,7 @@ export class ActiveExecutions {
 			this.activeExecutions[executionId].workflowExecution!.cancel();
 		}
 
-		return this.getPostExecutePromise(executionId);
+		return postExecutePromise;
```

Comment:
This was needed before I added the [`setTimeout` in WorkflowExecute](https://github.com/n8n-io/n8n/pull/6141/files#diff-0bfa6f1919bce33f0a8185d4496af9b38028eee2ed829ae01b225e2ff42a294bR832) because the executions was getting removed in the `.cancel` call.
reverted.